buildenv/jenkins/JenkinsfileBase

```diff
@@ -445,12 +445,12 @@ def setup() {
 			RESOLVED_MAKE = "if [ `uname` = AIX ] || [ `uname` = SunOS ] || [ `uname` = *BSD ]; then MAKE=gmake; else MAKE=make; fi"
 			
 			dir( WORKSPACE) {
-				// use sshagent with Jenkins credentials ID for all platforms except zOS
 				// on zOS use the user's ssh key
-				if (!env.SPEC.startsWith('zos')) {
```

Comment:
This and other `if` conditions were inverted so that we're always comparing positively. 
* It's easier for me to understand "if OS is zos then get_sources otherwise get_sources_with_authentication" than the other way around.
* In most of the cases I changed, there was a specific `if` and a specific` else` case; no fall-throughs, optional branches, or anything else.